packages/graphql-typescript-definitions/src/cli.ts

```diff
@@ -55,12 +54,16 @@ const argv = yargs
   })
   .help().argv;
 
+type Args = typeof argv;
+
 const builder = new Builder({
   cwd: argv.cwd,
-  schemaTypesPath: argv.schemaTypesPath,
-  addTypename: argv.addTypename,
-  enumFormat: argv.enumFormat,
-  customScalars: normalizeCustomScalars(argv.customScalars),
+  schemaTypesPath: argv.schemaTypesPath as Args['schema-types-path'],
```

Comment:
I think I prefer doing something like this instead.
```
const builder = new Builder({
  cwd: argv.cwd,
  schemaTypesPath: argv['schema-types-path'],
  addTypename: argv['add-typename'],
  enumFormat: argv['enum-format'],
  customScalars: normalizeCustomScalars(
    argv['custom-scalars'],
  ),
});
```

Any strong opinion?

Reply:
Yeah that's better.